doc/rados/operations/monitoring.rst

```diff
@@ -655,3 +655,84 @@ Runtime`_.
 
 .. _Viewing a Configuration at Runtime: ../../configuration/ceph-conf#viewing-a-configuration-at-runtime
 .. _Storage Capacity: ../../configuration/mon-config-ref#storage-capacity
+
+Messenger Status
+=================
+
+Ceph daemons and librados clients support an admin socket command
+``messenger dump`` that allows access to a snapshot of runtime
+information about connections, sockets, bound addresses, and kernel
+TCP stats (via tcp(7) TCP_INFO).
+
+.. note:: The queried messenger needs to lock the connection data
+  structures for the time it takes to create the snapshot. This might
+  interfere with normal operation. Use the ``dumpcontents`` argument
+  to limit data structures dumped.
+
+Examples
+---------
+
+Without specifying a messenger return the available messengers:
+
+.. prompt:: bash $
+
+   ceph tell osd.0 messenger dump
+
+.. code-block:: javascript
+
+ {
+    "messengers": [
+        "client",
+        "cluster",
+        "hb_back_client",
+        "hb_back_server",
+        "hb_front_client",
+        "hb_front_server",
+        "ms_objecter",
+        "temp_mon_client"
+    ]
+  }
+
+Client and cluster messenger correspond to the configured client /
+cluster network. Messengers with ``hb_`` prefix are part of the
```

Comment:
How does this look if there is no cluster/backend/replication/private network?

>prefixed with ``hb``

Reply:
The OSD always starts those messengers and binds them to the configured addresses. I think no cluster network basically means public == cluster network.

I'll add a ref to https://docs.ceph.com/en/reef/rados/configuration/network-config-ref/